framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1172,10 +1172,17 @@ protected void performFlowAnalysis(ClassTree classTree) {
 
                 while (!lambdaQueue.isEmpty()) {
                     Pair<LambdaExpressionTree, Store> lambdaPair = lambdaQueue.poll();
+                    Tree enclosingMethod =
+                            TreeUtils.enclosingOfKind(getPath(lambdaPair.first), Kind.METHOD);
+                    Tree enclosingClass =
+                            TreeUtils.enclosingOfKind(getPath(enclosingMethod), Kind.CLASS);
                     analyze(
                             queue,
                             lambdaQueue,
-                            new CFGLambda(lambdaPair.first),
+                            new CFGLambda(
+                                    lambdaPair.first,
+                                    (ClassTree) enclosingClass,
```

Comment:
Nit: I prefer giving the variables the more specific type rather than casting at the use -- that is, move these casts to lines 1175-1178.